os/include/tinyara/fs/fs.h

```diff
@@ -86,6 +86,11 @@
 #define CONFIG_MOUNT_POINT "/mnt/"
 #endif
 
+#ifdef CONFIG_FS_PROCFS
```

Comment:
Please split this change as a different commit.

Reply:
Done.